banzai/context.py

```diff
@@ -47,3 +47,6 @@ def __delattr__(self, item):
 
     def __setattr__(self, key, value):
         raise TypeError('Resetting attribute is not allowed. PipelineContext is immutable.')
+
```

Comment:
On line 24, change `preview_mode` keyword to `realtime_reduction`

Reply:
Ok. I guess so. I think this whole structure merits a little thought in a future PR.

Reply:
Done. I set the default to False. This also makes me think we should rethink this whole pattern.